packages/ui-keyring/src/Keyring.ts

```diff
@@ -185,6 +212,19 @@ export class Keyring extends Base implements KeyringStruct {
     this.rewriteKey(json, key, hexAddr, addressKey);
   }
 
+  private loadContract (json: KeyringJson, key: string) {
+    const address = this.encodeAddress(
+      isHex(json.address)
+        ? hexToU8a(json.address)
+        // FIXME Just for the transition period (ignoreChecksum)
+        : this.decodeAddress(json.address, true)
```

Comment:
Should be able to not have this here - since it is new and doesn't have old-encoded addresses floating around. (So don't need the FIXME, nor the `true`)